docs/style/entities/author.md

```diff
@@ -17,29 +17,30 @@ Try to create short, concise disambiguations.<br>
 Examples: "18th century novelist", "19th century Belgian playwright" or "Nigerian Africanfuturist writer"
 
 #### Group
-Use this for groups of Authors, like [collective pseudonyms](https://en.wikipedia.org/wiki/Category:Collective_pseudonyms) (pseudonyms used by multiple individuals), [house pseudonyms](https://en.wikipedia.org/wiki/Pen_name#Collective_names) (owned by a publisher and used by the writers working for them) and similar.
+Use this for groups of Authors, like [collective pseudonyms](https://en.wikipedia.org/wiki/Category:Collective_pseudonyms) (pseudonyms used by multiple individuals), [house pseudonyms](https://en.wikipedia.org/wiki/Pen_name#Collective_names) (owned by a publisher and used by the writers working for them) and similar.<br>
+Examples:<br>
+[Ellery Queen](https://bookbrainz.org/author/a2532a57-11fa-43d4-aa5d-c2892a2d0320), the group pseudonym of *Frederic Dannay* and *Manfred Bennington Lee*.<br>
+[James S. A. Corey](https://bookbrainz.org/author/c5c71242-f54c-4327-9510-a81064a9a852) pen name used by *Daniel Abraham* and *Ty Franck*.
```

Comment:
I just recently worked with a lettering studio in BB that is a group: [AndWorld Design](https://bookbrainz.org/author/9268a7af-2f90-49d8-91a1-9684620aaae9).